wcfsetup/install/files/lib/data/package/update/server/PackageUpdateServer.class.php

```diff
@@ -72,10 +72,10 @@ protected function handleData($data)
         }
 
         if ($this->isWoltLabUpdateServer()) {
-            $this->data['serverURL'] = "http://update.woltlab.com/{$prefix}{$officialPath}/";
+            $this->data['serverURL'] = "https://update.woltlab.com/{$prefix}{$officialPath}/";
```

Comment:
Just to make sure: This will enforce `https` for all 5.5 updates going forward. Previously the use of `https` was optional for 5.5 and earlier.